### PR TITLE
Replace the Documentation Makefile with a Shell Script

### PR DESCRIPTION
**Pull request by settermjd**
## Description

This PR does two things:

1. Ports the Makefile from the docs repository to a shell script, streamlined for a single-component Antora setup.
2. Adds support for exporting the manual to `.docx` format.

## Related Issue

n/a

## Motivation and Context

I've created this PR after making a similar change to the main docs repository and felt that it would make sense to make the change in this repository as well.

## How Has This Been Tested?

I used the script to build the different versions of the manual, as well as successfully ran all of the other functionality of the script.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by settermjd** [comment]
> Same here as on the android repo, I don't see any need for a .docx formatted documentation.

Just removed that in e6e83b4.

File: docs/books/ownCloud_iOS_App_Manual.adoc [hunk 1]
```diff
@@ -8,5 +8,4 @@ The ownCloud Team <docs@owncloud.com>
 :toclevels: 1
 :icons: font
 :icon-set: octicon
-
-include::modules/ROOT/pages/index.adoc[leveloffset=+1]
+:module_base_path: modules/ROOT/pages/
```
Comment (tboerger) [inline docs/books/ownCloud_iOS_App_Manual.adoc]: That should not be required?

Comment (settermjd) [reply docs/books/ownCloud_iOS_App_Manual.adoc]: It has now been removed. Can you review again, pls?

File: docs/books/ownCloud_iOS_App_Manual.adoc [hunk 2]
```diff
@@ -8,5 +8,3 @@ The ownCloud Team <docs@owncloud.com>
 :toclevels: 1
 :icons: font
 :icon-set: octicon
-
-include::modules/ROOT/pages/index.adoc[leveloffset=+1]
```
Comment (michaelstingl) [inline docs/books/ownCloud_iOS_App_Manual.adoc]: What gets removed here?

Comment (settermjd) [reply docs/books/ownCloud_iOS_App_Manual.adoc]: The content there is generated in the shell script.

**Pull request merged**